ophidian/util/Units.h

```diff
@@ -0,0 +1,107 @@
+#ifndef OPHIDIAN_UTIL_UNITS_H
+#define OPHIDIAN_UTIL_UNITS_H
+
+#include <ratio>
+#include <3rdparty/units/include/units.h>
+#include <boost/geometry/geometries/point_xy.hpp>
+
+namespace ophidian {
+namespace util {
+
+//Length Units
```

Comment:
I left it the way it was, but I can change it. units::length looks better, indeed.